cmake/usb/FindUSB.cmake

```diff
@@ -0,0 +1,115 @@
+if(NOT USB_FIND_COMPONENTS)
```

Comment:
Should the whole file go in stm32/FindUSB.cmake ,like the FindBSP, FindHAL, FindCMSIS? Or if we have another chip we will have both port in the same file?

Reply:
It could go there!